tools/harness-thci/OpenThread.py

```diff
@@ -1783,8 +1785,8 @@ def configBorderRouter(
             # TestHarness 1.2 converts 0x2001000000000000 to "2001000000000000"
             if P_Prefix is None:
                 P_Prefix = 0xfd007d037d037d03
-
             P_Prefix = '%016x' % P_Prefix
+
```

Comment:
Is the whitespace edits on this line and line 1786 above intentinoal?

Reply:
It was not intentional and I believe I fixed it.

Reply:
https://github.com/openthread/openthread/pull/7163/commits/63897904d8763d093a6b1ff157ca532b9cb28664

Reply:
The white space changes are still appearing in this diff.

Reply:
So sorry, I didn't see the second one.  I've now fixed that as well in https://github.com/openthread/openthread/pull/7163/commits/25217e9d0349089b5704e2ae4460dd3f600adad1, however, there is a check failing.  I don't believe my line removal should have caused that.  Is there something else going on here potentially?